src/main/kotlin/org/jitsi/jicofo/conference/colibri/ColibriSessionManager.kt

```diff
@@ -90,3 +87,13 @@ interface ColibriSessionManager {
         )
     }
 }
+
+data class ParticipantAllocationOptions(
```

Comment:
I might call this `ParticipantAllocationParameters` rather than `ParticipantAllocationOptions`, since it includes things like the various IDs that aren't optional.